src/app/shared/utils/hubConnectionStringHelper.ts

```diff
@@ -40,3 +41,23 @@ export const isValidEventHubConnectionString = (connectionString: string): boole
     const pattern = new RegExp('^Endpoint=sb://.*;SharedAccessKeyName=.*;SharedAccessKey=.*$');
     return pattern.test(connectionString);
 };
+
+export const getDaysBeforeHubConnectionStringExpires = (connectionStringWithExpiry: ConnectionStringWithExpiry) => {
+    // tslint:disable-next-line: no-magic-numbers
+    const millisecondsPerDay = 24 * 60 * 60 * 1000;
```

Comment:
Would be cleaner to do a single magic number here to avoid calculating every execution.

Reply:
sure